app/assets/stylesheets/govuk_publishing_components/components/_button.scss

```diff
@@ -22,6 +22,17 @@ $gem-quiet-button-hover-colour: darken($gem-quiet-button-colour, 5%);
   @include responsive-bottom-margin;
 }
 
+.gem-c-button--inline {
+  width: 100%;
+  margin-bottom: govuk-spacing(1);
+  margin-right: govuk-spacing(2);
```

Comment:
Isn't this supposed to be inherited from the `gem-c-button` margin?

Reply:
This is to add some spacing between the buttons on mobile, when they are stacked one above the other:

Without the margins:
<img width="254" alt="Screen Shot 2019-05-13 at 14 11 03" src="https://user-images.githubusercontent.com/29889908/57624045-fe106480-7588-11e9-89e2-ffd06cf687a9.png">

Reply:
Hmm.. I was hoping the button would have the default margin-bottom so we don't have to set it here, but I'm confusing the styles with the component in govuk-frontend